cmd/aasp/main.go

```diff
@@ -325,6 +363,19 @@ func main() {
 	outfile := flag.String("outfile", "", "The file to save the wrapped data")
 	flag.Parse()
 
+	var er error
+	bearerToken := ""
+	clientID := os.Getenv("AZURE_CLIENT_ID")
+	tenantID := os.Getenv("AZURE_TENANT_ID")
+	tokenFile := os.Getenv("AZURE_FEDERATED_TOKEN_FILE")
+	if tenantID != "" && tokenFile != "" {
+		bearerToken, er = getAccessTokenFromFederatedToken(context.TODO(), tokenFile, clientID, tenantID, "https://managedhsm.azure.net")
+		if er != nil {
+			log.Fatalf("Failed to get access token %v", er)
+		}
+	}
+	log.Printf("bearerToken: %v", bearerToken)
```

Comment:
Is there any reason that we want to print out the access token? I think we should just print out get access token succeeded or print out a redacted version of the token.

Reply:
Good point. That message is just for debugging.

Reply:
Actually line 366-377 are all for debugging purposes and should all be removed. @stevendongatmsft

Reply:
ok, I can remove it and test manually test again. The other day you said 
"We should merge it after we have a way to test it in confidential AKS, currently blocked by nested SNP"
Are we still blocked by nested SNP?